packages/suite/src/services/coinjoin/config.ts

```diff
@@ -88,6 +88,10 @@ export const DEFAULT_MAX_MINING_FEE = 3;
 // TODO: this value is supplied by the coordinator, but not saved yet; replace the hard-coded value with data fetched from coordinator
 export const UTXO_AMOUNT_THRESHOLD_FOR_COINJOIN = 5000;
 
+// coordinator fee rate from status format (0.003)
+// firmware format (0.003 * 10 * 8)
```

Comment:
nit: it's just a comment, but to prevent any confusion - there is missing one star between 10 * 8
```suggestion
// firmware format (300 000 =  0.003 * 10 ** 8)
```

Reply:
forcepushed